cmd/m3/set-password.go

```diff
@@ -101,7 +105,19 @@ func setPassword(ctx *cli.Context) error {
 	})
 
 	if err != nil {
-		log.Println(err)
+		if e, ok := status.FromError(err); ok {
+			switch e.Code() {
+			case codes.PermissionDenied:
```

Comment:
can you please add some comments for the permissionDenied? since this is only for the caller also aborted